processor/README.md

```diff
@@ -83,51 +83,10 @@ data cloning described in Exclusive Ownership section.
 The order processors specified in a pipeline is important as this is the
 order in which each processor is applied to traces and metrics.
 
-## <a name="attributes"></a>Attributes Processor
```

Comment:
This section was moved down so the diff makes it look like it was deleted. It wasnt!